website_sale_brand_catalogs/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Brand Catalogs for Website Sale",
+    "summary": """
+    This module manage the brands catalogs for products
+    """,
+    "category": "Website/Website",
+    "version": "15.0.1.0.0",
+    "author": "Vauxoo",
+    "license": "LGPL-3",
+    "depends": [
+        "crm",
+        "product_brand",
+        "website_sale",
+    ],
+    "demo": [
+        "demo/product_brand_demo.xml",
+        "demo/product_brand_attachment_demo.xml",
+    ],
+    "data": [
+        "security/ir.model.access.csv",
+        "data/ir_attachment_data.xml",
+        "data/res_config_settings_data.xml",
```

Comment:
This is an exception because settings files are not associated to a particular model, they for instance could contain `res.config.settings` for general settings and `base.install.language` to enable new languages.

```suggestion
        "data/settings.xml",
```